systemd-units/plymouth-poweroff.sh.in

```diff
@@ -2,4 +2,5 @@
 
 if [ "$1" = "poweroff" ]; then
 	@PLYMOUTH_CLIENT_DIR@/plymouth quit --retain-splash
+	sleep 1
```

Comment:
We may not really care, but on the other hand it would be great to get a better understanding of what's happening before setting the workaround in stone ;)